src/Base/PlacementPy.xml

```diff
@@ -63,14 +63,19 @@ Alias to move(), to be compatible with TopoShape.translate().\n
 vector : Base.Vector\n    Vector by which to move the placement.</UserDocu>
 			</Documentation>
 		</Methode>
-        <Methode Name="rotate">
+        <Methode Name="rotate" Keyword="true">
         <Documentation>
-            <UserDocu>rotate(center, axis, angle) -> None\n
+            <UserDocu>rotate(center, axis, angle, kwargs) -> None\n
 Rotate the current placement around center and axis with the given angle.
-This method is compatible with TopoShape.rotate().\n
+This method is compatible with TopoShape.rotate() if the (optional) keyword
+argument comp is True (default=False).
+
 center : Base.Vector, sequence of float\n    Rotation center.
 axis : Base.Vector, sequence of float\n    Rotation axis.
-angle : float\n    Rotation angle in degrees.</UserDocu>
+angle : float\n    Rotation angle in degrees.
+comp : bool\n    optional keyword only argument, if Ture (default=False),

```

Comment:
Typo: `Ture` should be `True`.

Reply:
fixed with d2d228dc89fcec750eb6226e64996dd3babe09a1

Reply:
There is an attempt to standardize the documentation, so the format should be
rotate(center, axis, angle, comp=False) -> None\n
In the argument description, please start it with capital, add four spaces in each line and limit the size (something like 80 characters) so it will look like:
```
angle : float
    Rotation angle in degrees.
comp : bool
    Optional keyword only argument, if True (default=False), behave like
    TopoShape.rotate() (i.e. the resulting placements are interchangeable).
```
And add the `</UserDocu>` tag to the end of the last string. Putting in the next line generate a blank line at the end of the tooltip.